test/unit/artefact/artefact_test.rb

```diff
@@ -0,0 +1,43 @@
+require_relative '../../test_helper'
+
+class Artefact::ArtefactTest < ActiveSupport::TestCase
+
+  context "taggable?" do
+
+    should "return true for artifacts not owned by publisher, smartanswers or testapp" do
+      Settings.expects(:untaggable_apps).returns(%w{ publisher smartanswers testapp }).at_least(1)
```

Comment:
I agree with using `context` in https://github.com/alphagov/panopticon/pull/314#discussion_r44527331, but here it wouldn't add much because "not owned by publisher, smartanswers or testapp" is not really a "precondition", but the actual thing were testing. Does that distinction make sense?